cmd/openshift-install/migrate.go

```diff
@@ -0,0 +1,186 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+
+	azmigrate "github.com/openshift/installer/pkg/migrate/azure"
+)
+
+var (
+	azureMigrateEligibleOpts struct {
+		public bool
+	}
+)
+
+func runMigrateAzureDNSEligibleCmd(cmd *cobra.Command, args []string) error {
+	return azmigrate.DoEligible(azureMigrateEligibleOpts.public)
+}
+
+func newMigrateAzureDNSEligibleCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "eligible",
+		Short: "Show legacy zones that are eligible to be migrated",
```

Comment:
would like to see long description also added for this.

Reply:
I've added long descriptions.They aren't a whole lot longer, but a bit more descriptive.